{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:03 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:03 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:03 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:03 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:03 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:03 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:03 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:03 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:03 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:03 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:03 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:03 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:04 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:04 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:04 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:04 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:04 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:04 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:04 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:04 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:04 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:04 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:04 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:04 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:04 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:04 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"9:45:04 AM"}
{"level":"error","message":"(node:15724) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): MongoError: E11000 duplicate key error collection: lm-api-qa.sync_contacts index: Sync_Contact_Id_1 dup key: { : null }","timestamp":"9:45:04 AM"}
{"level":"error","message":"(node:15724) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): MongoError: Index with name: Posp_Id_1 already exists with different options","timestamp":"9:45:04 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"9:45:29 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"9:45:29 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"9:45:32 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"9:45:41 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"9:45:43 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"9:53:10 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"9:53:10 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:07:21 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:07:21 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:07:25 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:07:25 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:42:44 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:42:44 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:42:47 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:42:47 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:43:12 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:43:13 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:43:18 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:43:18 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:44:02 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:44:03 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:46:05 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:46:06 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:46:33 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:46:33 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:48:34 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:48:35 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:48:37 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:48:38 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:48:41 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:48:42 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:48:55 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:48:56 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:49:08 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:49:08 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:49:42 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:49:42 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:49:50 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:49:51 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:49:55 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:49:56 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:51:05 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:51:05 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:59:01 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:59:01 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:59:02 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:01:50 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:02:55 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:09:34 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:09:45 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:21:23 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:21:24 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:26:06 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:28:29 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:29:34 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:29:43 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:35:21 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:35:54 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:35:54 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:35:56 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:42:29 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:42:45 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:43:29 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:25:54 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:27:07 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:27:08 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:27:22 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:30:03 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:31:44 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"1:32:18 PM"}
{"level":"error","message":"(node:6336) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): MongoError: E11000 duplicate key error collection: lm-api-qa.sync_contacts index: Sync_Contact_Id_1 dup key: { : null }","timestamp":"1:32:18 PM"}
{"level":"error","message":"(node:6336) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): MongoError: Index with name: Posp_Id_1 already exists with different options","timestamp":"1:32:18 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:32:45 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:33:19 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:34:17 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:34:18 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:34:25 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:34:27 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:34:31 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:34:31 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"1:34:35 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:34:36 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:34:52 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:34:54 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:34:57 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:34:57 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"1:35:03 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:35:04 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:35:28 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:35:29 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:36:28 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:36:28 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:55:37 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"1:55:37 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:00:37 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:00:38 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"2:00:56 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:00:56 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:01:11 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:01:14 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:01:14 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"2:01:39 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:01:40 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:08:19 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:08:19 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"2:08:25 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:08:25 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:08:35 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:08:38 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:08:38 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"2:08:47 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:08:48 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"2:09:02 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:09:02 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:08 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:10 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:11 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:11 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:11 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:11 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:11 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:12 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:12 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:12 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:13 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:13 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:13 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:14 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:14 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:14 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:14 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:14 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:15 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:15 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:15 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:15 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:15 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:15 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:15 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:15 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"2:24:15 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:31:30 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:31:51 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"2:32:22 PM"}
{"level":"error","message":"(node:3752) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): MongoError: E11000 duplicate key error collection: lm-api-qa.sync_contacts index: Sync_Contact_Id_1 dup key: { : null }","timestamp":"2:32:22 PM"}
{"level":"error","message":"(node:3752) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): MongoError: Index with name: Posp_Id_1 already exists with different options","timestamp":"2:32:22 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:32:25 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:32:45 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:33:12 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"2:33:17 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"2:33:17 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:33:18 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:36:08 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:36:19 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:36:37 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:36:43 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:37:23 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:46:18 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:46:23 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:46:33 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:46:56 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:47:09 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:47:09 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"4:36:12 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"4:36:19 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"4:36:21 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"4:36:23 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"4:36:31 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"4:36:31 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"4:36:36 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"4:36:37 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"5:48:12 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"5:48:12 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"5:48:13 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"5:48:22 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"5:48:51 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"5:49:05 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"5:51:04 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"5:51:24 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"5:51:29 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"5:52:29 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"5:52:29 PM"}
